src/styles/main.scss

```diff
@@ -1 +1,70 @@
-// put code here
+$clock-size: 80vmin;
+
+@mixin center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@mixin hands-position {
+  transform-origin: 50% 100%;
+  transform: rotate(0deg);
+}
```

Comment:
move it to separate files

Reply:
Clock face was tested on 1920x1080 screen only 😂, it's only border and that was half intentional. Will do


> move it to separate files

Should I do it even if such small projects? Or is it as a habit/sake of task?